### DISCUSSION TITLE
what should i do if a json is ERR_BLOCKED_BY_ADMINISTRATOR?

### DISCUSSION THREAD

**Discussion by Numbernu897** [3kh0/ext-remover#653] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
im currently trying to do ltbeef and in the faq doesnt seem to talk about being blocked by administrator, only by client

i tried to do the exploit on the blocked page anyways and yet i can still open the url just fine without being unblocked

**Comment by RavenChill-Dev** [comment, marked as answer]
Idk.... Try using crosh
